docsite/rst/guide_vagrant.rst

```diff
@@ -27,54 +28,48 @@ Vagrant Setup
 
 The first step once you've installed Vagrant is to create a ``Vagrantfile``
 and customize it to suit your needs. This is covered in detail in the Vagrant
-documentation, but here is a quick example:
+documentation, but here is a quick example that includes a section to use the
+Ansible provisioner to manage a single machine:
 
-.. code-block:: bash
+.. code-block:: ruby
```

Comment:
the `Vagranfile` uses [Ruby syntax](https://docs.vagrantup.com/v2/vagrantfile/index.html) indeed.